eox_tenant/middleware.py

```diff
@@ -126,3 +93,17 @@ def process_request(self, request):
                     {'domain': domain, }
                 )
             )
+
+
+class EoxTenantCurrentSiteMiddleware(MiddlewareMixin):
```

Comment:
What is you opinion on leaving the name CurrentSiteMiddleware since this is supposed to be a drop in replacement.
In common.py it will still be super clear, because of the path not being `django.contrib...` but `eox_tenant....`